actions.md

```diff
@@ -0,0 +1,547 @@
+```
+
+A URL with the following format will be returned for standard actions:
+```
+ok: got action actionName
+https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/greeting
+```
+
+**[[NH: check link]]**
+Authentication is required when invoking an action via an HTTPS request using this resource path. For more information regarding action invocations using the REST interface, see [Using REST APIs with OpenWhisk](rest_api.md#actions).
+
+**[[NH: check link:]]**
+Another way of invoking an action that does not require authentication is via [web actions](webactions.md#web-actions).
+
+Any action may be exposed as a web action, using the `--web true` command line option when creating or updating an action.
+
+```
+nim action update greeting --web true
+ok: updated action greeting
+```
+
+The resource URL for a web action is different:
+```
+nim action get greeting --url
+ok: got action greeting
+https://${APIHOST}/api/v1/web/${NAMESPACE}/${PACKAGE}/greeting
+```
+
+You can use `curl` or `wget` to invoke the action.
+```
+curl `nim action get greeting --url | tail -1`.json
+{
+  "payload": "Hello, Toto from somewhere!"
+}
+```
+
+### Save action code
+
+Code associated with an existing action may be retrieved and saved locally. Saving can be performed on all actions except sequences and docker actions.
+
+1. Save action code to a filename that corresponds with an existing action name in the current working directory. A file extension that corresponds to the action kind is used. An extension of _.zip_ is used for action code that is a zip file.
+
+  **[[NH: please check command and output:]]**
+  ```
+  nim action get /whisk-system/samples/greeting --save
+  ok: saved action code to /path/to/namespace/greeting.js
+  ```
+
+2. You can provide your own file name and extension by using the `--save-as` flag.
+
+  **[[NH: check output:]]**
+  ```
+  wsk action get /whisk-system/samples/greeting --save-as hello.js
+  ok: saved action code to /path/to/namespace/hello.js
+  ```
+
+### List actions
+
+You can list all the actions that you have created using the following commnand:
+
+```
+nim action list
+  actions
+  /guest/mySequence                  private sequence
+  /guest/greeting                    private nodejs:6
+```
+
+Here, we see actions listed in order from most to least recently updated. For easier browsing, you can use the flag `--name-sort` or `-n` to sort the list alphabetically:
+
+**[[NH: Output list seems backwards for alphabetical order?]]**
+```
+nim action list --name-sort
+  actions
+  /guest/mySequence                  private sequence
+  /guest/greeting                    private nodejs:6
+```
+
+The list is now sorted alphabetically by namespace, then package name if any, and finally action name, with the default package (no specified package) listed at the top. **[[NH: Is namespace listed in this output example? What does output look like if there's a package? Does nim output list the namespace?]]**
```

Comment:
For your reference, this is what our output looks like in the latest version (not yet fully public).   I am working to update our public release so you will have the latest code to work with.
```
 nim action list --name-sort
Datetime          Version   Access      Kind           Actions                                           
04/01/20 10:04:60 0.0.5     web open    nodejs:10      /wbtestni-grinjpsjnuh/action                      
03/31/20 14:03:26 0.0.4     web open    nodejs:10      /wbtestni-grinjpsjnuh/billing/awsbill             
03/21/20 16:03:10 0.0.6     web open    php:7.3        /wbtestni-grinjpsjnuh/visits/counter              
03/31/20 14:03:30 0.0.4     web open    nodejs:10      /wbtestni-grinjpsjnuh/billing/datadogbill         
03/31/20 14:03:31 0.0.4     web open    nodejs:10      /wbtestni-grinjpsjnuh/billing/dobill              
03/22/20 11:03:44 0.0.8     private     nodejs:10      /wbtestni-grinjpsjnuh/echo/echo                   
03/23/20 13:03:76 0.0.8     private     php:7.3        /wbtestni-grinjpsjnuh/visits/info                 ```

Reply:
note ive revised this even further but think i am done with those changes.

Reply:
Copied this in for now, bretter than what was there.